distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: Implementation of the user-defined currency
 - Feature: Add OpenGL drawing engine.
```

Comment:
I agree with Maarten. Could you put OpenGL at the top and the custom currency below it?